routes/web.php

```diff
@@ -34,3 +34,17 @@
 Route::get('login/google/callback', [LoginController::class, 'handleGoogleCallback']);
 
 Route::post('/logout', [LoginController::class, 'logout'])->name('logout');
+
+Route::middleware(['auth:admin'])->group(function () {
+    // Routes only accessible to admins
+    Route::get('/admin/home', [\App\Http\Controllers\AdminController::class, 'home'])->name('admin.home');
```

Comment:
Please uniform when calling controller path

Reply:
same as [LoginController::class, 'redirectToFacebook']

Reply:
This is unused
web.php
use App\Http\Controllers\Auth\LoginController;
use App\Http\Controllers\Auth\RegisterController;
use App\Http\Controllers\HomeController;

Reply:
also category route please change